### PR TITLE
sfwbar: init at 1.0_beta11

### LINKED ISSUE #231995: Package request: sfwbar
**Project description**
SFWBar (S* Floating Window Bar) is a flexible taskbar application for wayland compositors, designed with a stacking layout in mind.

**Metadata**

* homepage URL:https://github.com/LBCrion/sfwbar
* source URL:https://github.com/LBCrion/sfwbar/archive/refs/tags/v1.0_beta11.tar.gz
* license: GPL-3.0
* platforms: unix, linux

### PR DESCRIPTION
**Pull request by NotAShelf**
###### Description of changes

Adds [sfwbar](https://github.com/LBCrion/sfwbar)

Closes #231995 

P.S. my previous PRs had issues because I'm clearly a git expert /s

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by geri1701** [comment]
Result of `nixpkgs-review pr 232069` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sfwbar</li>
  </ul>
</details>

**Review by geri1701 — APPROVED** [review]

**Comment by geri1701** [comment]
I guess the `v` should be removed from the PR title please.

File: pkgs/applications/misc/sfwbar/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+, pkg-config
+, gtk-layer-shell
+, libpulseaudio
+, libmpdclient
+, libxkbcommon
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "sfwbar";
+  version = "1.0_beta11";
```
Comment (tomodachi94) [inline pkgs/applications/misc/sfwbar/default.nix:17]: Is there any particular reason for using this beta release?

Comment (NotAShelf) [reply pkgs/applications/misc/sfwbar/default.nix:17]: Yes, there are no proper stable releases worth using - I believe the maintainer treats beta releases akin to actual releases.

**Review by tomodachi94 — APPROVED** [review]
Lgtm. Thanks for your contribution!

File: pkgs/applications/misc/sfwbar/default.nix (hunk 1 above)
Comment (tomodachi94) [reply pkgs/applications/misc/sfwbar/default.nix:17]: Ah, got it. Feel free to mark this as resolved; I was just curious.

**Pull request merged**